acceptancetests/assess_log_rotation.py

```diff
@@ -158,9 +158,9 @@ def check_expected_backup(key, logprefix, action_output):
             (log_name, backup_pattern))
 
     size = int(log["size"])
-    if size < 299 or size > 301:
+    if size > 30:
```

Comment:
As an aside, it could be worth checking both the compressed version, then decompressing it and checking the uncompressed version.. this also has the advantage of ensuring that the log format is understood and decompressable.